### PR TITLE
Us004 load events

### PR DESCRIPTION
**Pull request by madeye-matt**
Added a loader script for loading collection exercise events and added test data received from Lou

### CODE REVIEW

**Review by andrewmil — CHANGES_REQUESTED** [review]
We should probably also update `collex-loader/Readme.md` It's not totally obvious how to run it/if it needs to be run separately/together with `load.py`

**Comment by madeye-matt** [comment]
@andrewmil README.md updated as requested

File: collex-loader/load_events.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    try:
+        raw = datetime.strptime(date, '%d%m%y')
+        raw = raw.replace(tzinfo=timezone.utc)
+    except ValueError:
+        print("Failed to parse {}".format(date))
+        raise
+
+    time_str = raw.isoformat(timespec='milliseconds')
+    // WATCH OUT: this removes the last : from a valid ISO8601 date string to form another
```
Comment (andrewmil) [inline collex-loader/load_events.py]: Wrong way to comment in python

**Review by tej99 — APPROVED** [review]
LG 👍

File: collex-loader/load_events.py (hunk 1 above)
Comment (madeye-matt) [reply collex-loader/load_events.py]: Ooops! *looks shifty* fixed now

**Review by andrewmil — APPROVED** [review]

**Comment by benjefferies** [comment]
We will need to update the Makefile in https://github.com/ONSdigital/ras-integration-tests too

**Pull request merged**